compiler/src/iree/compiler/Dialect/Flow/Transforms/OutlineDispatchRegions.cpp

```diff
@@ -27,6 +30,75 @@ namespace IREE {
 namespace Flow {
 namespace {
 
+// Estimates the evaluation cost of a linalg op using a heuristic cost model.
+static int estimateLinalgOpCost(linalg::LinalgOp op) {
+  if (op.hasDynamicShape()) {
+    // Note: bounded dynamic shapes would be interesting, if the compiler used
+    // them. For now just treat dynamic shapes as arbitrarily large.
+    return INT_MAX;
+  }
+
+  int cost = 1;
```

Comment:
Using uint64_t might be more explicit and system-independent.

Reply:
Changed to `int64_t` matching what `getStaticLoopRanges()` returns, and considering I use `-1` as the initial value.